Websites/webkit.org/wp-content/themes/webkit/build-archives.php

```diff
@@ -23,7 +23,7 @@ public static function object() {
     }
 
     private function call ($endpoint, $params = array()) {
-        $url = add_query_arg($params, 'https://q1tzqfy48e.execute-api.us-west-2.amazonaws.com/v3/' . $endpoint);
```

Comment:
I also see the old URL in `Tools/WebKitArchiveSupport/AWS-APIGateway/stage-v2.json`.

Reply:
Thanks for pointing it out. That's the actual AWS-APIGateway codes which is no longer in use (v3 now) and also should be removed. I will open a different bug for that.